keboliu/sonic-buildimage
# [Mellanox] install sdk libs to pmon

**- What I did**
Install SDK libs to pmon image
mount sdk shared stuff to pmon container
remove unused '-e' options for Mellanox syncd container 

**- How I did it**
inject pmon container image dependency on SDK libs
modify image ctrl templates for mount SDK volumes to pmon and remove unused "-e" option 

**- How to verify it**
run sdk APIs in pmon can see it works:

```
root@mtbc-sonic-03-2700:~# docker exec -it pmon sx_api_ports_dump.py 
=================================================================================================================
|  log_port|local_port|label_port|       mtu| admin_s|  oper_s|       module_s|  pvid|     oper_speed|  fec_mode|
=================================================================================================================
|   0x10100|         1|        17|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x10300|         3|        18|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x10500|         5|        19|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x10700|         7|        20|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x10900|         9|        21|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x10b00|        11|        22|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x10d00|        13|        23|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x10f00|        15|        24|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x11100|        17|        25|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x11300|        19|        26|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x11500|        21|        27|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x11700|        23|        28|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x11900|        25|        29|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x11b00|        27|        30|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x11d00|        29|        31|      9122|      UP|      UP|        PLUGGED|     1|       50GB_CR2|        RS|
|   0x11f00|        31|        32|      9122|      UP|      UP|        PLUGGED|     1|       50GB_CR2|        RS|
|   0x12100|        33|        15|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x12300|        35|        16|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x12500|        37|        13|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x12700|        39|        14|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x12900|        41|        11|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x12b00|        43|        12|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x12d00|        45|         9|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x12f00|        47|        10|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x13100|        49|         7|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x13300|        51|         8|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x13500|        53|         5|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x13700|        55|         6|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x13900|        57|         3|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x13b00|        59|         4|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x13d00|        61|         1|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
|   0x13f00|        63|         2|      9122|      UP|      UP|        PLUGGED|     1|      100GB_CR4|        RS|
=================================================================================================================
```

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
